app/views/layouts/_right_menu.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if current_user %>
+  <% if current_user.email == 'sieki13@o2.pl' %>
```

Comment:
Now your view has knowledge how exactly the admin is to be distinguished from regular user.

The view should be interested in whether the user is an admin or not, but it should not know anything more. The view should be interested only in rendering things. No other logic should be placed in the view.

Reply:
Yes, I have added user role. Now view only checks if user is admin, I think it looks better.